tembo-operator/yaml/sample-message-queue.yaml

```diff
@@ -73,6 +73,8 @@ spec:
           - total_messages:
               description: Total number of messages that have passed into the queue.
               usage: GAUGE
+        target_databases:
```

Comment:
Do we also need to add this to the Stack definition at src/stacks/templates?

Reply:
No it will add it automatically when reconciling the configmap.